ang/civicase/CaseOverview.html

```diff
@@ -3,16 +3,39 @@
     <div class="civicase__case-overview-container" civicase-custom-scrollbar scrollbar-config="{autoHide: false}">
       <div class="civicase__case-overview__flow">
         <div class="civicase__case-overview__flow-status">
+          <civicase-popover
+            class="civicase__case-overview__flow-status-settings"
```

Comment:
What is reasoning behind the naming of this class? Should not it be Case Status Settings?

Reply:
@deb1990 the "flow" prefix was suggested by @AkA84 here:
https://github.com/compucorp/uk.co.compucorp.civicase/pull/89

tbh, I'm not 100% sure what they are referring to with "flow", but I'm guessing they are referring to how the status progress from "open" to "completed" and anything in between.

If we change this we'll need to change a lot of other places too and before doing so we should discuss with @AkA84.